app/services/HandleWorker.php

```diff
@@ -0,0 +1,70 @@
+<?php
+
+namespace GanttDashboard\App\Services;
+
+use GanttDashboard\App\Models\Workers;
+use Phalcon\Session\Adapter\Files;
+use Phalcon\Security;
+
+class HandleWorker
+{
+    /**
+     * @var Files
+     */
+    private $session;
+
+    /**
+     * @var Security
+     */
+    private $security;
+
+    /**
+     * @var Workers
+     */
+    private $worker;
+
+    /**
+     * HandleWorker constructor.
+     * @param Files $sessionService
+     * @param Security $securityService
+     * @param Workers $worker
+     */
+    public function __construct(
+        Files $sessionService,
+        Security $securityService,
+        Workers $worker
+    ) {
+        $this->session = $sessionService;
```

Comment:
name you class members correct: sessionService, securityService...
Workers $worker => singular plural???
I needed to look to see what is a Workers class... apparently this is a model. Make this clear from naming.